test/Microsoft.Azure.Webjobs.Extensions.Storage.UnitTests/Blobs/Listeners/ScanBlobScanLogHybridPollingStrategyTests.cs

```diff
@@ -146,15 +147,60 @@ public async Task BlobPolling_IgnoresClockSkew()
             timeMap[expectedNames.Single()] = now.AddSeconds(-60);
             RunExecuterWithExpectedBlobs(expectedNames, product, executor);
 
-            expectedNames.Clear();
+            expectedNames.Add(await CreateBlobAndUploadToContainer(container));
+            timeMap[expectedNames.Last()] = now.AddSeconds(-59);
+
+            // We should see the new item. We'll see 2 blobs, but only process 1 (due to receipt).
+            RunExecuterWithExpectedBlobs(expectedNames, product, executor, 1);
+
+            Assert.Equal(2, container.CallCount);
+            Assert.Equal(expectedNames, executor.BlobReceipts);
+        }
+
+        [Fact]
+        public async Task BlobPolling_IncludesPreviousBatch()
```

Comment:
This test failed before the change. We'd miss that second blob with the same timestamp.